apps/docs/docs/fundamentals/customization.mdx

```diff
@@ -128,6 +128,66 @@ Check out the full source code [here](https://github.com/MarceloPrado/flash-cale
 
 Sometimes, you need more than just changing styles. You might want to change the layout, add new components, or even remove some of them. In this case, you can easily build your own calendar using the base Calendar components from [above](#anatomy).
 
+You can override the `textProps` to control the nested [Text components](https://reactnative.dev/docs/text). For example [disabling font scaling](https://reactnative.dev/docs/text#allowfontscaling) for accessibility on the Calendar's Day:
```

Comment:
Looks great! One last change: instead of adding inside the "Composing your own calendar", let's add afterwards, in a new `### Changing the text props` section! Afterwards, I'll merge it!